include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -4298,6 +4298,13 @@ struct OrtApi {
   void(ORT_API_CALL* ReleaseROCMProviderOptions)(_Frees_ptr_opt_ OrtROCMProviderOptions* input);
 
   /// @}
+
+  /** \brief Create and register Cuda Allocator
+   *  \param[in] env OrtEnv instance
+   *  \param[in] mem_info OrtMemoryInfo instance
```

Comment:
the description is not complete. It will also be helpful to add how this is different than CreateAndRegisterAllocator